pkg/k8s/apis/cilium.io/v2/cec_types.go

```diff
@@ -88,7 +88,7 @@ type Service struct {
 	// In CiliumEnvoyConfig namespace defaults to the namespace of the CEC,
 	// In CiliumClusterwideEnvoyConfig namespace defaults to "default".
 	// +kubebuilder:validation:Optional
-	Namespace string `json:"namespace"`
+	Namespace string `json:"namespace,omitempty"`
```

Comment:
Technically, this is a breaking change. What's the reason to add this option?

Reply:
there is no particular reason, I think it's making sense to have omitempty for optional field. 

I am not 100% if it's actually breaking change, but let me just drop this commit.

Reply:
> I am not 100% if it's actually breaking change

I think it is. If some events consumer expect the field to always be present and act on it, they will break once it's not present anymore. But then again, I'm not exactly sure which consumer that would be.